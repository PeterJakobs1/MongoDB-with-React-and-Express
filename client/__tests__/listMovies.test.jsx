describe("listMovies component", () => {
  it("Shows loading screen", () => {});
});
