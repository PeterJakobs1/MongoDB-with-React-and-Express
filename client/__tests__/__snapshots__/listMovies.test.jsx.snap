// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listMovies component Shows loading screen 1`] = `"<div>Loading...</div>"`;
